api_app/analyzers_manager/observable_analyzers/bgp_ranking.py

```diff
@@ -0,0 +1,144 @@
+import json
+import logging
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class BGPRanking(classes.ObservableAnalyzer):
+    """
+    wrapper for https://github.com/D4-project/BGP-Ranking
+    """
+
+    base_url: str = "https://bgpranking-ng.circl.lu"
+
+    period: int  # optional
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        final_response = {}
+
+        # get ASN from ip
+
+        logger.info(f"Extracting ASN from IP: {self.observable_name}")
+        response = requests.get(
+            self.base_url + "/ipasn_history/?ip=" + self.observable_name
+        )
+        response.raise_for_status()
```

Comment:
I would like to add another parameter that we usually add when interacting with free services: `timeout`.
That would be the timeout for the requests. If we don't set it, these requests can keep going for a long time and this analyzer can become the bottleneck of a playbook. This could be really a big problem.

The default should be 5 seconds, like in other analyzers. Thank you